package.json

```diff
@@ -3,9 +3,11 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
-    "@centrifuge/gateway-server": "file:packages/server",
     "@centrifuge/gateway-lib": "file:packages/lib",
-    "@centrifuge/gateway-ui": "file:packages/ui"
+    "@centrifuge/gateway-server": "file:packages/server",
+    "@centrifuge/gateway-ui": "file:packages/ui",
+    "bignumber.js": "^9.0.0",
```

Comment:
Not sure if these should go in the server package instead.

Reply:
I could not find where bignumber.js is used. I think bn.js is used